src/singletons/Settings.hpp

```diff
@@ -375,6 +375,14 @@ class Settings : public ABSettings, public ConcurrentSettings
         "/misc/twitch/messageHistoryLimit",
         800,
     };
+    IntSetting twitchMessageLogLimit = {
+        "/misc/twitch/messageLogLimit",
+        8000,
+    };
```

Comment:
The setting controls the amount of scrollback in the Usercard logs
The setting does not do anything Twitch-specific
Setting name should be changed to reflect this
Variable name should be changed to reflect this
Default value should be revererted back to the default 1000

Reply:
I think the default for the log history should be increased here as in faster chats 1000 isn't enough. For example at 10-20 message per second chat would allow for around 13-10 minutes worth of history with the 8000 limit. With 1000 it is less than a minute for user logs.

Reply:
This is the reason it's configurable - if you run into the issue where messages disappear from the buffer/scrollback too quickly you can change it. For the average user, this change wouldn't change anything except for the memory usage